xclim/data/fr.json

```
@@ -201,305 +201,304 @@
+    "title": "Nombre de jours où la température quotidienne est au-dessus du 90e centile",
+    "abstract": "Nombre de jours où la température quotidienne est au-dessus du 90e centile."
   },
   "TN_MIN": {
-    "long_name": "Minimum de la température journalière",
-    "description": "Minimum {freq:m} de la température journalière minimale.",
-    "title": "Température journalière minimale",
-    "abstract": "Température journalière minimale."
+    "long_name": "Minimum de la température minimale quotidienne",
+    "description": "Minimum {freq:m} de la température minimale quotidienne.",
+    "title": "Température minimale",
+    "abstract": "Minimum de la température minimale quotidienne."
   },
   "TN_MAX": {
-    "long_name": "Maximum de la température journalière minimale",
-    "description": "Maximum {freq:m} de la température journalière minimale.",
+    "long_name": "Maximum de la température minimale quotidienne",
+    "description": "Maximum {freq:m} de la température minimale quotidienne.",
     "title": "Maximum de la température minimale",
-    "abstract": "Maximum de la température journalière minimale."
+    "abstract": "Maximum de la température minimale quotidienne."
   },
   "TN_MEAN": {
-    "long_name": "Moyenne de la température journalière minimale",
-    "description": "Moyenne {freq:f} de la température journalière minimale.",
+    "long_name": "Moyenne de la température minimale quotidienne",
+    "description": "Moyenne {freq:f} de la température minimale quotidienne.",
     "title": "Moyenne de la température minimale",
-    "abstract": "Moyenne de la température journalière minimale."
+    "abstract": "Moyenne de la température minimale quotidienne."
   },
   "TN10P": {
-    "long_name": "Nombre de jours où la température minimale est en dessous du {per_base_thresh}e percentile",
-    "description": "Nombre {freq:m} de jours où la température minimale est en dessous du {per_base_thresh}e percentile. Le {per_base_thresh}e percentile est calculé à partir d'une fenêtre de {per_window} jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence.",
-    "title": "Nombre de jours où la température minimale est en dessous du {per_base_thresh}e percentile",
-    "abstract": "Nombre de jours où la température minimale est en dessous du {per_base_thresh}e percentile."
+    "long_name": "Nombre de jours où la température minimale est sous du 10e centile",
+    "description": "Nombre {freq:m} de jours où la température minimale est sous du {per_base_thresh}e centile. Le {per_base_thresh}e centile est calculé à partir d'une fenêtre de {per_window} jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence.",
+    "title": "Nombre de jours où la température minimale est sous du 10e centile",
+    "abstract": "Nombre de jours où la température minimale est sous du 10e centile."
```

```suggestion
    "long_name": "Nombre de jours ayant une température minimale sous le 10ᵉ centile",
    "description": "Nombre {freq:m} de jours où la température minimale est sous le {per_base_thresh}ᵉ centile. Le {per_base_thresh}ᵉ centile est calculé à partir d'une fenêtre de {per_window} jours centrée sur chaque jour du calendrier à l'intérieur d'une période de référence.",
    "title": "Nombre de jours ayant une température minimale sous le 10ᵉ centile",
    "abstract": "Nombre de jours où la température minimale est sous le 10ᵉ centile."
```
Notez qu'ici on a omis le "quotidienne" qu'on avait dans les nombre de jours plus haut. Je préfère ça.